ocaml/testsuite/tests/typing-missing-cmi-indirections/test.ml

```diff
@@ -14,5 +14,3 @@ module = "client.ml"
 ocamlc_byte_exit_status = "2"
 ****** check-ocamlc.byte-output
 *)
-
-(* CR layouts v2.9: error message here is unreviewed *)
```

Comment:
I don't like this error message because it doesn't give any suggestion of how these two comments connect:
  * "the layout of alias must be a sublayout of value"
  * "because the type argument of list has layout value"

I think this could be improved if the error message gave a location for the second bullet point, namely the RHS of `and indirect_usage = alias list`.

============

More context

`test.compilers.reference`:

```
File "client.ml", line 2, characters 0-19:
2 | and alias = missing
    ^^^^^^^^^^^^^^^^^^^
Error: 
       The layout of alias is any, because
         the .cmi file for Missing.t is missing.
       But the layout of alias must be a sublayout of value, because
         the type argument of list has layout value.
       No .cmi file found containing Missing.t.
       Hint: Adding "missing" to your dependencies might help.
```

The recursive group is:

```
type missing = Present.t
and alias = missing
and indirect_usage = alias list
```

Reply:
filed ticket